armotypes/pagination_structs.go

```diff
@@ -182,6 +182,10 @@ type UniqueValuesResponseV2 struct {
 	// UniqueTotalCount map[string]ElasticRespTotal `json:"totalCount"`
 }
 
+type DeleteCountResponse struct {
```

Comment:
Sice this response is only for mongo maybe we should put in under portalTypes.go ?